### DISCUSSION TITLE
redis with ssl but don't use client certificate

### DISCUSSION THREAD

**Discussion by seunggyu0572** [redis/lettuce#2137] [Q&A]
I had a question about ssl options.
I set tls-client-auth no in redis.conf.
but how to set in spring boot ??
I couldn't find client auth options in lettuceConnectionFactory.

**Comment by mp911de** [comment]
You need to configure `SslOptions` with the appropriate key- and trustmanager settings and provide these details through `ClientOptions` and `LettuceClientConfiguration`.
